cmd/dockerd/daemon.go

```diff
@@ -16,6 +16,7 @@ import (
 
 	"github.com/container-orchestrated-devices/container-device-interface/pkg/cdi"
 	containerddefaults "github.com/containerd/containerd/defaults"
+	"github.com/containerd/containerd/log"
```

Comment:
```suggestion
	containerdlog "github.com/containerd/containerd/log"
```

To make it more clear where these constants come from.